frontend/src/concepts/k8s/NameDescriptionField.tsx

```diff
@@ -12,6 +26,14 @@ type NameDescriptionFieldProps = {
   autoFocusName?: boolean;
   showK8sName?: boolean;
   disableK8sName?: boolean;
+  nameHelperText?: string;
+  isNameValid?: boolean;
```

Comment:
I think you don't need these 2 fields anymore, because you will add the helper text inside the component.

Reply:
cc @uidoyen